vico/compose/src/main/java/com/patrykandpatrick/vico/compose/legend/Legends.kt

```diff
@@ -62,14 +63,44 @@ public fun verticalLegend(
  * @param labelText the text content of the label.
  */
 @Composable
-public fun verticalLegendItem(
+public fun LegendItem(
```

Comment:
As per Jetpack Compose convention, composable function that returns something should start with a lowercase letter.
```suggestion
public fun legendItem(
```